StickyTabBarViewController/Classes/MainTabBarController.swift

```diff
@@ -10,53 +10,52 @@ import UIKit
 public protocol StickyViewControllerSupporting: UITabBarController {
     var collapsedHeight: CGFloat { get set }
     var animationDuration: TimeInterval { get set }
-    func configureCollapsedViewController(withChildViewController childViewController: Expandable)
-    func removeCollapsibleViewController(animated: Bool)
-    func collapseCollapsibleViewController()
+    func configureCollapsableChild(_ childViewController: Expandable)
+    func removeCollapsableChild(animated: Bool)
+    func collapseChild()
 }
 
 open class StickyViewControllerSupportingTabBarController: UITabBarController, StickyViewControllerSupporting {
-    private var collapsableVCFlow: ExpandableViewController?
+    
+    // MARK: - Public properties
     public var collapsedHeight: CGFloat = 50.0
     public var animationDuration: TimeInterval = 0.5
     
-    final public func updateCollapsedHeight(to value: CGFloat) {
-        collapsedHeight = value
-    }
+    // MARK: - Private properties
+    private var collapsableVCFlow: ExpandableViewController?
     
-    final  public func updateAnimationDuration(to value: TimeInterval) {
-        animationDuration = value
-    }
+    // MARK: - Public API
     
-    final public func configureCollapsedViewController(withChildViewController childViewController: Expandable) {
+    /// Prepares View Controller to be embedded as a child (wrapped in another internal View Controller)
+    /// - Parameter childViewController: A View Controller conforming to `Expandable` protocol
+    final public func configureCollapsableChild(_ childViewController: Expandable) {
         guard collapsableVCFlow == nil else {
             return
         }
         childViewController.loadView()
-        childViewController.expander = self
+        childViewController.container = self
         collapsableVCFlow = ExpandableViewController(withChildVC: childViewController,
                                                      collapsedHeight: collapsedHeight,
-                                                     animationDuration: animationDuration,
-                                                     minimisedView: childViewController.minimisedView)
-        guard let collapsableVCFlow = collapsableVCFlow else {
-            return
-        }
-        collapsableVCFlow.tabController = self
-        view.addSubview(collapsableVCFlow.view)
-        addChild(collapsableVCFlow)
-        collapsableVCFlow.view.translatesAutoresizingMaskIntoConstraints = false
-        collapsableVCFlow.view.leftAnchor.constraint(equalTo: view.leftAnchor).isActive = true
-        collapsableVCFlow.view.rightAnchor.constraint(equalTo: view.rightAnchor).isActive = true
+                                                     animationDuration: animationDuration)
+        
+        collapsableVCFlow!.tabController = self
```

Comment:
I know this can be a debatable change as I am using now IUO but I favour less code over impossible scenarios. 
In this case `collapsableVCFlow` has just been initialised so it's not possible for it to not exist.  

Just because the definition is optional it does not mean we have to add unnecessary checks where it's not needed.

Reply:
I actually previously also implemented it this way, but since Im used to have swiftlint for so long, couldnt be happy with force unwrapping and changed it 😄But yes, this is a valid use case for it indeed.

Reply:
Haha, yeah, I know the feeling. I'm still not sure I get much value out of linting honestly, I feel that most of the time it just adds burden to the developer. 

Formatting is another thing & I like it. Have you consider using a code formatting tool? 
I use `SwiftFormat` in personal projects & I'm pretty happy with it 

https://github.com/Thurman1776-/Uncomplect-us/blob/master/Mac-App/BuildTools/Package.swift

Reply:
Yeah 😅This one looks cool! We can add this to this project as well.